packages/synchro-charts/src/components/sc-dial/utils/svgBuildPath.ts

```diff
@@ -0,0 +1,128 @@
+const mathSin = Math.sin;
```

Comment:
is there some library that does this for us?

Reply:
This is a base function with no third party libraries.

Reply:
Sorry, I didn't quite understand. How can I help you with it? Can you please further explain your requirement?

Reply:
if there is a library we can utilize to make these curved svg corners instead of owning a bunch of code around curves, that would be ideal - as long as it doesn't require bundling a large dependency that adds to the size of the distributed code

Reply:
did you consider https://github.com/d3/d3-shape/blob/main/README.md#curves ?

Reply:
Sorry, I encountered a test problem that could not be solved for a while, so I could not submit the code today.
The problem is  that run `yarn test` to get `Jest encountered an unexpected token` error , because import { arc } from 'd3-shape'.

Reply:
Done